ccd-definition/AuthorisationCaseField/AuthorisationCaseField-COSC-nonprod.json

```diff
@@ -22,5 +22,41 @@
         "CRUD": "CRU"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL${CCD_DEF_VERSION}",
```

Comment:
These all changes (declaration, permissions) related to new fields should be in files with name including "JO" extension?